spec.html

```diff
@@ -30522,6 +30522,7 @@
             1. Let _index_ be _O_.[[ArrayIteratorNextIndex]].
             1. Let _itemKind_ be _O_.[[ArrayIterationKind]].
             1. If _a_ has a [[TypedArrayName]] internal slot, then
+              1. Perform ? ValidateTypedArray(_a_).
```

Comment:
that goes all the way back to the discussion on https://github.com/tc39/ecma262/issues/713#issuecomment-256087125 where this should be performing ValidateTypedArray or not.